gcp/config/jiant_paths.sh

```diff
@@ -14,6 +14,7 @@ export JIANT_PROJECT_PREFIX="$HOME/exp"
 export ELMO_SRC_DIR="/nfs/jiant/share/elmo"
 # cache for BERT etc. models
 export HUGGINGFACE_TRANSFORMERS_CACHE="/nfs/jiant/share/transformers_cache"
+export PYTORCH_TRANSFORMERS_CACHE="/nfs/jiant/share/transformers_cache"
```

Comment:
Why not just use one? I think the latter replaces the former.

Reply:
The latter is used by Transformers, while the former is used by jiant.
(In theory we should supply the former directly to Transformers which means the latter should not be necessary, but I ran into errors until I supplied it, so I'll look into that.)

Reply:
Dang.